mobile/lib/features/wallet/send/send_onchain_screen.dart

```diff
@@ -62,11 +64,21 @@ class _SendOnChainScreenState extends State<SendOnChainScreen> {
 
     setState(() {
       _feeEstimates = fees;
-      _amount = widget.destination.amount;
-      _controller.text = _amount.formatted();
+      Amount amt = widget.destination.amount;
+      amt = amt.sats == 0 ? Amount(1000) : amt;
+      logger.i(amt);
```

Comment:
I guess the logger is not needed anymore.

Reply:
Good catch, thanks!